.github/workflows/publish-charts.yaml

```diff
@@ -8,7 +8,7 @@ on:
 # Prevent two release workflows from running concurrently.
 concurrency: 
   group: chart-releaser
-  cancel-in-progress: true
+  cancel-in-progress: false
 
 jobs:
   release:
```

Comment:
Unrelated to this PR (maybe) but I would drop the accursed releaser action and instead use `helm push` to an OCI registry.

Reply:
Unrelated to this PR. I fact, I still don't know if I am going to be able to integrate the release toolkit in this PR.